recipes/sexpdata/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "sexpdata" %}
+{% set version = "0.0.3" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "1ac827a616c5e87ebb60fd6686fb86f8a166938c645f4089d92de3ffbdd494e0" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - sexpdata
+
+about:
+  home: https://github.com/tkf/sexpdata
+  license: BSD
+  license_family: BSD
+  summary: S-expression parser for Python
+  description: "S-expression parser for Python\n==============================\n\n`sexpdata` is a simple S-expression parser/serializer.  It has\nsimple `load` and `dump` functions like `pickle`, `json`\
```

Comment:
Would it be possible to remove the control characters?

Reply:
I'm not sure what you mean here. The `\n`s are there because the string has newlines.

Reply:
Yes, the description is rather difficult to read with those in. I'm also not certain how they will render when this gets converted to a feedstock.